src/AndersonAcceleration.f90

```diff
@@ -6,55 +6,54 @@
 ! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
 ! can be found in LICENSE.txt in the head directory of this repository.        !
 !++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
-!> @brief Module provides an interface to Anderson Acceleration
-!>
-!> Currently supported TPLs include:
-!>  - Trilinos
-!>
-!> @par EXAMPLES
-!> @code
-!>
-!> @endcode
-!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief A Fortran 2003 module that encompasses the native implementation of
+!>        Anderson Acceleration in Futility.
 MODULE AndersonAccelerationTypes
```

Comment:
A pretty minor one, but our style guide says modules should be named "<filename without extension>Module"... so for this, "AndersonAccelerationModule".  Granted, virtually nothing in Futility has been updated to follow this, but since this module is unused in downstream code so far, let's try to start it off correctly.

Reply:
I'm glad you mentioned this. I was leaning toward changing it, but since the module already existed before I gutted and remade it I didn't want to change it without a second opinion.